hpfennig.new.rst

```diff
@@ -0,0 +1,164 @@
+------------------------------------------------------------
+summary
+------------------------------------------------------------
+
+.. class:: myname
+
+Herbert Pfennig
+
+.. class:: contact
+
+**E:** dasein555@gmail.com
+**L:** Redwood City, CA
+
+.. class:: summary
+
+Internet handyman with over 14 years of experience working in startups. I
+enjoy solving problems and building infrastructure to help engineers get
+stuff done through automation. Over the years I have built up experience in
+Test Automation, Systems and Network Administration, Internal Tools
+Development and Release Engineering.
+
+------------------------------------------------------------
+skillset
+------------------------------------------------------------
+
+| **OS:**             Linux, OSX, FreeBSD
+| **Platforms:**      Docker, AWS, GCE, Kubernetes, Virtualbox, VMWare
+| **Languages:**      Python, Shell scripting, with some GO, C, C++ etc.
+| **Internet:**       HTTP, SMTP, DNS, DHCP, GPXE, TCP/IP, UDP
+| **Tools:**          Git, Gerrit, Github Enterprise, Jenkins, GNUmake, BSDMake,
+                      Ansible, Terraform, Packer, Sumologic, Datadog, Quay
+| **Learning:**       Concourse CI, GO
+
+------------
+employment
+------------
+
+.. class:: jobtitle
+..
+
+	*[07/2014 - Current]* :: **Release\Tools Engineer** :: `Bracket Computing`_
+
+	.. _Bracket Computing: https://brkt.com/
+
+.. class:: notes
+..
+
+	Acquired by VMware Inc. in 4/2018
+
+.. class:: tools
+..
+
+	Tools: aws, gce, docker, kubernetes, jenkins, git, gerrit, packer
+
+* Migrated company build infrastructure to use a Docker based build pipeline
+* Developed and rolled out the continuous delivery pipeline for hosted
+  infrastructure running in AWS on Kubernetes
+* Implemented a Git precommit workflow with Gerrit and Jenkins for all
+  internal repos to ensure code review and code quality consistency
+* TODO (hpfennig) One more thing about AWS and GCE skillset
+
+.. class:: jobtitle
+..
+
+	*[02/2012 - 07/2014]* :: **Principal Release Engineer** :: `Silvertail Systems Inc.`_
+
+	.. _Silvertail Systems Inc.: http://www.emc.com/domains/silvertail/index.htm
+
+.. class:: notes
+..
+
+	Acquired by EMC Inc. in 2/2012
+
+.. class:: tools
+..
+
+	Tools: unittest2, jenkins, buildbot, git, github enterprise, rpmbuild, make, gpxe, vsphere, redis, c++, gcc
+
+* Owned the build, test and rpm-based release pipeline for product written in C++ and Python
```

Comment:
There is no proof Capthook ever existed...since it is proprietary code. I am hoping the general nature of the bullet points invites discussion where I can talk about more detail...